content/developers/api-reference/assets-api/index.md

```diff
@@ -109,6 +109,78 @@ All Public Assets are then given a read-only public URL that can be retrieved us
 This link can be shared with anyone to give them read-only access to the Asset or Event without the need to sign in.
 
 To interact with the unauthenticated Public Interface for a Public Asset see the [Public Assets API Reference](../public-assets-api/). To update the Assets and Events as the creating Tenant on a Public Asset's authenticated Private Interface, you would still use the standard Assets and Events API as normal.
+
+#### Creating a Document Profile Asset
+This class of Asset conforms to the [Document Profile Developer Pattern](/developers/developer-patterns/document-profile/), which allows you to trace the lifecyle of a document.
+
+Define the asset parameters and store in `/path/to/jsonfile`:
+
+```json
+{
+    "attributes": {
+        "arc_description":"Test Document",
+        "arc_display_type":"Marketing glossy",
+        "arc_display_name":"Test Document Profile Asset",
+        "arc_profile":"Document",
+        "document_hash_value":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+        "document_hash_alg":"sha256",
+        "document_version":"1",
+        "document_status":"Published",
+        "some_custom_attribute":"anything you like"
+    },
+    "chain_id":"",
+    "behaviours": [
+        "Builtin",
+        "RecordEvidence"
+    ],
+    "proof_mechanism":"SIMPLE_HASH"
+}
+```
+{{< note >}}
+**Note**: Document Profile Assets must be set to `public` to be compatible with [Instaproof](/platform/overview/instaproof/) verification.
+{{< /note >}}
+
+Create the Asset:
+
+```bash
+curl -v -X POST \
+    -H "@$BEARER_TOKEN_FILE" \
```

Comment:
Can we update this with `-H "@rkvst-bearer.txt" \`, so it's consistent with PR #669

Reply:
Done